### PR TITLE
Clean up the framework

### PR DESCRIPTION
**Pull request by FeGeyer**
- Removed old architecture files
- Deleted unused helper functions
- Deleted unused layers
- Removed unused loss functions
- Deleted `hooks.py`

**Still to do:**

- Write docstrings
- Look into simulation and evaluation files

### CODE REVIEW

**Comment by FeGeyer** [comment]
Also updated the version of GitHub actions used

File: radionets/evaluation/utils.py [hunk 1]
```diff
@@ -267,10 +256,6 @@ def get_images(test_ds, num_images, norm_path="none", rand=False):
     if rand:
         indices = torch.randint(0, len(test_ds), size=(num_images,))
     img_test = test_ds[indices][0]
-    norm = "none"
-    if norm_path != "none":
-        norm = pd.read_csv(norm_path)
-    img_test = do_normalisation(img_test, norm)
     img_true = test_ds[indices][1]
     if len(img_true.shape) == 3:
```
Comment (ArnePoggenpohl) [inline radionets/evaluation/utils.py:260]: Do we still need that? I would have to squeeze again, when using a list with len(list.shape)==3

Comment (FeGeyer) [reply radionets/evaluation/utils.py:260]: Why does this have an effect on squeezing? This only applies a no longer supported normalisation?

Comment (ArnePoggenpohl) [reply radionets/evaluation/utils.py:260]: I was only talking about 260 and 261. My list is of shape [batch size, num. of component, attributes], thus the length is equal 3. And when opening an image with that function, I get an extra axis cause of this unsqueeze.

Comment (FeGeyer) [reply radionets/evaluation/utils.py:260]: Have to have another look into https://github.com/radionets-project/radionets/pull/116/files

**Review by ArnePoggenpohl — APPROVED** [review]

**Pull request merged**